pointing_component/sal3.9_dev/setup.sh

```diff
@@ -6,17 +6,22 @@ echo "#"
 echo "# Setting up and starting ATPtg CSC."
 echo "#"
 
-source scl_source enable devtoolset-3
 source /opt/lsst/software/stack/loadLSST.bash
-setup lsst_distrib
 source $LSST_SDK_INSTALL/setup.env
-setup ts_sal -t $USER
 
 while :
     do
+    
+    if [ "$TELESCOPE" == "Main" ]; then
+	echo "# Running main telescope pointing component..."
+	/home/saluser/repos/ts_pointing_common/build/bin/MTPtgController
+	echo "# Main telescope pointing component exited."
+    else
+	echo "# Running auxiliary telescope pointing component..."
         /home/saluser/repos/ts_pointing_common/build/bin/ATPtgController
-        echo "# ATPtg exited. Restarting..."
+	echo "# Main telescope pointing component exited."
```

Comment:
Technically should be Auxiliary telescope

Reply:
There's and good catch! thanks!